css/properties/custom-property.json

```diff
@@ -112,6 +112,109 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "safe-area-inset": {
+            "__compat": {
+              "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/env()",
+              "spec_url": "https://drafts.csswg.org/css-env/#safe-area-insets",
+              "description": "Safe area inset variables <code>safe-area-inset-top</code>, <code>safe-area-inset-right</code>, <code>safe-area-inset-bottom</code>, and <code>safe-area-inset-left</code>",
+              "support": {
+                "chrome": {
+                  "version_added": "69"
+                },
+                "chrome_android": {
+                  "version_added": "69"
+                },
+                "edge": {
+                  "version_added": "79"
+                },
+                "firefox": {
+                  "version_added": "65"
+                },
+                "firefox_android": {
+                  "version_added": "65"
+                },
+                "ie": {
+                  "version_added": false
+                },
+                "opera": {
+                  "version_added": "56"
+                },
+                "opera_android": {
+                  "version_added": "48"
+                },
+                "safari": {
+                  "version_added": "11"
+                },
+                "safari_ios": {
+                  "version_added": "11"
+                },
+                "samsunginternet_android": {
+                  "version_added": "10.0"
+                },
+                "webview_android": {
+                  "version_added": "69"
+                }
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "titlebar-area": {
+            "__compat": {
+              "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/env()",
+              "spec_url": "https://wicg.github.io/window-controls-overlay/#title-bar-area-env-variables",
+              "description": "Window Controls Overlay variables <code>titlebar-area-x</code>, <code>titlebar-area-y</code>, <code>titlebar-area-width</code>, and <code>titlebar-area-height</code>",
+              "support": {
+                "chrome": {
+                  "version_added": "92",
+                  "notes": "Linux support was added in 93."
```

Comment:
OS limitations like this require `partial_implementation`. See https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#operating-system-limitations-imply-partial_implementation In this case it means that you'll need two support statements:

1. From Version 93, no notes or partial.
2. From version 92 to 93, with a note and partial.

Also, typically, notes are written in the present tense (as if from the perspective of the version(s) to which it applies). Something like this: "Before version 93, Linux is unsupported."

Reply:
Awesome. Thanks for the tip. Now I know how to do this properly.